src-tauri/src/app_config.rs

```diff
@@ -20,6 +20,7 @@ pub struct AppConfigFromFile {
     pub last_binaries_update_timestamp: SystemTime,
     pub allow_analytics: bool,
     pub anon_id: String,
+    pub log_mining_activity_permission: bool,
```

Comment:
there is already a config: "allow_analytics"

Reply:
Sorry, I thought it's for something else